index.html

```diff
@@ -7,75 +7,108 @@
   <link rel="stylesheet" href="styles/main.css" />
 </head>
 <body>
-  <div class="container">
-    <div class="jumbotron" style="color:#F0F4F7">
-      <h1 class="title1 padded" align="center">Save the Bay</h1>
-    </div>
 
-    <div class="row">
-      <p class="col-md-12 text-justify lead"><br />
-        Save The Bay is the largest regional organization working to protect, restore and celebrate San Francisco Bay since 1961. We engage more than 50,000 supporters, advocates, and volunteers to protect the Bay and make it cleaner and healthier for people and wildlife.
-      </p>
-    </div>
+  <nav class="navbar navbar-default">
+    <div class="container-fluid">
+      <!-- Brand and toggle get grouped for better mobile display -->
+      <div class="navbar-header">
+        <button type="button" class="navbar-toggle collapsed" data-toggle="collapse" data-target="#bs-example-navbar-collapse-1" aria-expanded="false">
+          <span class="sr-only">Toggle navigation</span>
+          <span class="icon-bar"></span>
+          <span class="icon-bar"></span>
+          <span class="icon-bar"></span>
+        </button>
+        <a class="navbar-brand" href="#!">Save The Bay</a>
+      </div>
 
-    <div class="row">
-      <p class="col-md-12 text-justify">
-        Save The Bay is leading a region-wide effort to re-establish 100,000 acres of tidal marsh that are essential for a healthy Bay. Volunteers from the community, local businesses, and schools work with our science team to perform hands-on restoration of the Bay shoreline. Save The Bay inspires the next generation of Bay activists through our award-winning restoration education programs. 
-      </p>
-    </div>
+      <!-- Collect the nav links, forms, and other content for toggling -->
+      <div class="collapse navbar-collapse" id="bs-example-navbar-collapse-1">
+        <ul class="nav navbar-nav">
+          <li class="active"><a href="#!">Link <span class="sr-only">(current)</span></a></li>
+          <li><a href="#!">Link</a></li>
+        </li>
+      </ul>
+      <ul class="nav navbar-nav navbar-right">
+        <li>
+          <a href="blog.html">Blog</a></li>
+        </li>
+        <li>
+          <a href="#!">Contact Us</a></li>
+        </li>
+      </ul>
+    </div><!-- /.navbar-collapse -->
+  </div><!-- /.container-fluid -->
+</nav>
 
-    <div class="row">
-      <p class="col-md-12 text-justify">
-        For more than half a century, Save The Bay has given San Francisco Bay a voice and shown that Bay Area residents can work together to make positive changes in their communities.
-        <br /><br />
-      </p>
-    </div>
+<div class="container">
+  <div class="jumbotron" style="color:#F0F4F7">
+    <h1 class="title1 padded" align="center">Save the Bay</h1>
+  </div>
+
+  <div class="row">
+    <p class="col-md-12 text-justify lead"><br />
```

Comment:
We should avoid the use of `<br />` for padding, and style the containing element instead using `padding-top`.  This provides us a lot more control over the layout (since `br` can be styled independently, it could potentially affect this `p` adversely).